src/guc.c

```diff
@@ -745,6 +746,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_bool_compression"),
+							 "Enable bool compression functionality",
+							 "Enable bool compression",
+							 &ts_guc_enable_bool_compression,
+							 true,
+							 PGC_USERSET,
+							 0,
+							 NULL,
+							 NULL,
+							 NULL);
+
```

Comment:
Let's say it's experimental and disable by default for now. I think we need to play with it a little more to understand if we're ready to freeze the on-disk format. After we release it officially, it won't be possible to make changes.

Reply:
> Let's say it's experimental

In the GUC description I mean.